### ISSUE TITLE
[Feature Request] Option To Disable Proxies Per Indexer

### ISSUE THREAD

**Issue by zero77** [Jackett/Jackett#2539]
Can you please add the option to disable proxies on specific indexers although, enable proxy support for all indexers by default like normal.

Because, sometimes adding a proxy will break some indexers but, is needed to fix other indexers.
Thanks

**Comment by edalex86** [comment]
Nice idea!
Hope to see it implemented.

**Comment by roguesync** [comment]
Would be even better if each indexer could get its own proxy config.

Something like:

* Global proxy option
* Exempt specific tracker from global proxy
* Provide proxy for a specific tracker

**Comment by MonkWho** [comment]
Please add my vote to "per indexer proxy config".

Currently I have to run two instances of jackett because one of my indexers has to be accessed via .onion from my region. So I have a VM running tor as a service with a port open to internal connections. One instance of jackett does not have proxy configured and the other instance has tor VM ip and port set in proxy settings. It would be great not to have to run two instances of jackett and have an option to configure proxy on an indexer level.

An option to bypass the proxy per indexer would also work but is a bit less flexible. However I would think this option would probably be much easier to implement.

**Comment by Chris230291** [comment]
Yes please. Most trackers will not work with a proxy for me so I go without.
Most of the remaining ones seem to work fine with my Tor proxy, so I'd like to enable it only on those.

**Comment by AARON8888** [comment]
Hi all! Any advantage with this issue? I think not yet, right?

**Comment by ilike2burnthing** [comment]
The issue is still open, so no.

**Comment by AARON8888** [comment]
I appreciate someone could work about this issue, even though next year. :)

**Comment by AARON8888** [comment]
I guess this matter is not easy to be implemented in Jackett (Core, C#, ...), but it will be very useful everyone who use proxy like me and others and want to use proxy connection in some "special" indexers.

Maybe is it not very difficult to use an address exception to be added in each indexer in Jackett UI website in case if needed. This idea is available in Windows 10, 11 in Proxy section.  For instance: if I write this address *.yggtorrent.* in proxy exception box, my proxy in my system will not run at any url of yggtorrent site.

**Comment by AARON8888** [comment]
Hi @ilike2burnthing!
Sorry to disturb you.
In UI/Dashboard there is a field named: "External access". What for this option if I mark it? Is it related with proxy option?
--
Another thing. As proxy option can be choosen in UI generally, can be implemented a field to write excluded url (tracker url site) so that proxy cannot be applied. Maybe it will be easier than to implement a disablement option Proxy Per Indexer.

What do you think?

--
I´m very interested in adding some indexers in UI and exclude them of proxy use. 
95% of my indexers use proxy option included into UI.

**Comment by garfield69** [comment]
>In UI/Dashboard there is a field named: "External access". What for this option if I mark it? Is it related with proxy option?

this option is for when you want to open access via LAN/WAN, instead of the default access for local only (localhost/127.0.01
IF you do enable external access, we recommend setting an admin password for security.

**Comment by ilike2burnthing** [comment]
Either this or https://github.com/Jackett/Jackett/issues/11693 would need to be implemented to achieve per indexer or per domain proxy support. Currently, you'd need to either run 2 instances of Jackett, or set up an external PAC proxy and configure Jackett to use that.

**Comment by AARON8888** [comment]
Thanks for all info @garfield69 and @ilike2burnthing.
I think "run 2 intances" will be the best solution in my case. 
I´m using Chrome for UI with proxy settings in an user profile in Windows 10.
How to use second instance of UI in a second user profile in Chrome in Windows 10? Is it possible?

**Comment by ilike2burnthing** [comment]
No, you'd need to run two instances of Jackett itself, not just access Jackett using two different browser profiles. Easiest way to do this is with Docker.

**Comment by garfield69** [comment]
and don't forget to change the port on the docker jackett from 9117 to another so you can access the correct dashboard
